MonoGame.Framework/Matrix.cs

```diff
@@ -162,10 +259,13 @@ public struct Matrix : IEquatable<Matrix>
 		                                            0f, 1f, 0f, 0f, 
 		                                            0f, 0f, 1f, 0f, 
 		                                            0f, 0f, 0f, 1f);
-        #endregion Private Members
+        #endregion
 
         #region Public Properties
-        
+
+        /// <summary>
+        /// Gets or sets the backward vector of this matrix.

```

Comment:
We discussed this one before...  no `Gets or sets`...  that is obvious from it being a property.

Also no `of this matrix` as of course it is of this matrix... what other matrix would it be?

If we want to add more words... lets add useful info like:  

`The backward vector formed from the third row M31, M32, M33 elements.`.